server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -509,6 +530,25 @@ export class RestGitService {
 		}
 	}
 
+	private async getCache<T>(key: string): Promise<T | undefined> {
+		if (this.cache) {
+			// Attempt to cache to Redis - log any errors but don't fail
+			const cachedValue: T | undefined = await runWithRetry(
+				async () => this.cache.get<T | undefined>(key),
+				"RestGitService.getCache",
+				3,
+				1000,
+				winston,
```

Comment:
Maybe we can add comments to indicate what each of these parameters mean?
Also, what parameter relates to this `winston` line? Is that `telemetryProperties`? I'm confused about why we would pass winston here - shouldn't this be `this.lumberProperties`?

Reply:
Good question! I just copy-pasted from the `setCache` code's use of runWithRetry 😆 I'll take a deeper look

Reply:
Got it! Then maybe that has been a bug in setCache all along 😄